packages/state-manager/src/types.ts

```diff
@@ -18,9 +17,14 @@ import {
   type CommunityMetadata,
   type PermsData,
   type UserProfile,
+  type DeleteChannelResponse,
 } from '@quiet/types'
 
-type EmitEvent<Payload> = (payload: Payload) => void
+interface EventsMap {
+  [event: string]: (...args: any[]) => void
+}
```

Comment:
The new socket.io doesn't seem to export `DefaultEventsMap` anymore and I found an issue where they suggested making our own types. So here I've copied the `DefaultEventsMap` implementation.